aw_client/client.py

```diff
@@ -33,31 +35,17 @@ def __init__(self, client_name: str, testing=False):
 
         # Setup failed queues dir
         self.data_dir = appdirs.user_data_dir("aw-client")
```

Comment:
Not an issue in your changes, but while you're at it you should take a look at the new `aw_core.dirs` where we have a wrapper for the data/conf/cache-dir getters so we only depend on appdirs there (note the new folder structure enforced by the code in that module).